### PR TITLE
AzureCP PR - Update Compute API Version to 2016-03-30

### PR DESCRIPTION
**Pull request by huangpf**
The content to be added to Changelog is as follows:
* AzureCP PR - Update Compute API Version to 2016-03-30

### CODE REVIEW

File: lib/commands/arm/vm/vmStorageProfile._js [hunk 1]
```diff
@@ -232,12 +232,12 @@ __.extend(VMStorageProfile.prototype, {
         },
 
         _parseOSDiskParams: function(params, createOption, _) {
-          if (createOption !== 'Attach' && createOption !== 'FromImage') {
```
Comment (yugangw-msft) [inline lib/commands/arm/vm/vmStorageProfile._js]: consider use case insensitive comparison exposed from utils.js

**Comment by yugangw-msft** [comment]
a nit, the rest looks good to me

File: lib/commands/arm/vm/vmStorageProfile._js (hunk 1 above)
Comment (huangpf) [inline lib/commands/arm/vm/vmStorageProfile._js]: They have to match exactly. Re: https://github.com/Azure/azure-sdk-for-node/issues/1658

**Comment by huangpf** [comment]
There is a regression about the enum parsing in new Node SDK. Close it for now.

**Comment by huangpf** [comment]
@yugangw-msft @amarzavery 
Could you please help take a look again? I changed the client version to ```0.16.1```. Thanks.

**Pull request merged**